includes/polyfills/class-polyfills.php

```diff
@@ -0,0 +1,110 @@
+<?php
+/**
+ * Polyfills.
+ *
+ * @package Newspack
+ */
+
+namespace Newspack;
+
+use \WP_Error;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Manages settings for Polyfills.
+ */
+class Polyfills {
```

Comment:
WDYT calling this Amp_Polyfills?

Reply:
Good idea, done in 59706c845288c3aca1518871289741c8bd6a59b0